Tangem/Modules/Markets/TokenMarketsDetails/Subviews/MarketsPortfolioContainerView/MarketsPortfolioTokenItemView.swift

```diff
@@ -136,78 +111,72 @@ struct MarketsPortfolioTokenItemView: View {
+            actionType.icon.image
                 .renderingMode(.template)
-        }
-    }
-}
+                .resizable()
+                .frame(size: .init(bothDimensions: 20))
+                .foregroundStyle(Colors.Icon.primary1)
+                .padding(6)
+                .background(
+                    Circle()
+                        .fill(Colors.Background.tertiary)
+                )
+                .padding(.vertical, 3)
+                .padding(.horizontal, 2)
 
-// MARK: - Initialization
-
-extension MarketsPortfolioTokenItemView {
-    @available(iOS 16.0, *)
-    init(
-        viewModel: MarketsPortfolioTokenItemViewModel,
-        cornerRadius: CGFloat,
-        roundedCornersVerticalEdge: RoundedCornersVerticalEdge?
-    ) {
-        self.viewModel = viewModel
-        previewContentShapeCornerRadius = cornerRadius
-
-        switch roundedCornersVerticalEdge {
-        case .topEdge:
-            roundedCornersConfiguration = RoundedCornersConfiguration(
-                topLeadingRadius: cornerRadius,
-                topTrailingRadius: cornerRadius
-            )
-        case .bottomEdge:
-            roundedCornersConfiguration = RoundedCornersConfiguration(
-                bottomLeadingRadius: cornerRadius,
-                bottomTrailingRadius: cornerRadius
-            )
-        case .all:
-            roundedCornersConfiguration = RoundedCornersConfiguration(
-                topLeadingRadius: cornerRadius,
-                bottomLeadingRadius: cornerRadius,
-                bottomTrailingRadius: cornerRadius,
-                topTrailingRadius: cornerRadius
-            )
-        case .none:
-            roundedCornersConfiguration = nil
+            VStack(alignment: .leading, spacing: 2) {
+                Text(actionType.title)
+                    .style(Fonts.Bold.subheadline, color: Colors.Text.primary1)
+
+                if let description = actionType.description {
+                    Text(description)
+                        .multilineTextAlignment(.leading)
+                        .style(Fonts.Regular.caption1, color: Colors.Text.tertiary)
+                }
+            }
+
+            Spacer()
         }
     }
 
-    @available(iOS, obsoleted: 16.0, message: "Use 'init(viewModel:cornerRadius:roundedCornersConfiguration:)' instead")
-    init(
-        viewModel: MarketsPortfolioTokenItemViewModel,
-        cornerRadius: CGFloat
-    ) {
-        self.viewModel = viewModel
-        previewContentShapeCornerRadius = cornerRadius
-        roundedCornersConfiguration = RoundedCornersConfiguration()
+    private func makeLineRowActionItem() -> some View {
+        HStack(alignment: .center) {
+            Spacer()
+                .frame(width: 18)
+
+            Rectangle()
+                .fill(Colors.Stroke.primary)
+                .frame(width: 1, height: 16)
+                .padding(.vertical, 4)
+
+            Spacer()
+        }
+        .frame(maxWidth: .infinity)
```

Comment:
вообще выглядит так, что можно сделать:
```suggestion
            Rectangle()
                .fill(Colors.Stroke.primary)
                .frame(width: 1, height: 16)
                .padding(.vertical, 4)
                .padding(.leading, 18)
```

Reply:
а ещё выглядит так, будто эта полоска должна быть длинней, если текст описания действия на 2 строки, т.к. отступы начинают варьироваться. Или это обсуждали и черточка всегда фиксированной высоты?

Reply:
> а ещё выглядит так, будто эта полоска должна быть длинней, если текст описания действия на 2 строки, т.к. отступы начинают варьироваться. Или это обсуждали и черточка всегда фиксированной высоты?

Черточка же фиксированной высоты

Reply:
она фиксированная, потому что в дизайне не указан вариант с двухстрочной текстовкой, когда отступы вокруг иконки становятся больше